server/local.ts

```diff
@@ -23,7 +23,7 @@ export function handler(env: Env): Handler {
           return await provider.handle(request, pathInfo)
         } catch (e) {
           if (e instanceof Response) return e
-          if (env.dbg) console.error(e)
+          console.error(e)
```

Comment:
We can re-add the toggling on/off when we tackle a comprehensive approach to logging from the Capi server (perhaps using `Logger` from `deno_std`). I plan to take a pass at this in a PR that also addresses #649.